cmd/access/unstaked_access_node_builder.go

```diff
@@ -92,20 +120,12 @@ func (builder *UnstakedAccessNodeBuilder) enqueueUnstakedNetworkInit(ctx context
 		// for now we use the empty metrics NoopCollector till we have defined the new unstaked network metrics
 		unstakedNetworkMetrics := metrics.NewNoopCollector()
```

Comment:
Can we remove this now that it is not used below?

Reply:
they are sill being used in the `initMiddleware` and `initNetwork`